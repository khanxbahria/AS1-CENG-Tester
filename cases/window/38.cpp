#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->print();
Tab U = Tab("aaa", "ff","hhh");
wind->getActiveTab();
Tab D = Tab("cc", "ttt","p");
wind->newTab(D);
Tab Y = Tab("kk", "mmm","u");
wind->newTab(U);
Tab F = Tab("z", "s","bbb");
wind->print();
Tab W = Tab("s", "vvv","vv");
wind->getActiveTab();
wind->print();
Tab T = Tab("pp", "qq","yy");
wind->isEmpty();
Tab S = Tab("aa", "v","nnn");
wind->moveActiveTabTo(3);
Tab H = Tab("g", "jj","sss");
wind->changeActiveTabTo(2);
wind->changeActiveTabTo(2);
Tab N = Tab("bb", "nnn","uu");
wind->closeTab();
Tab J = Tab("oo", "z","n");
wind->moveActiveTabTo(1);
wind->closeTab();
Tab M = Tab("j", "www","qq");
wind->isEmpty();
wind->changeActiveTabTo(4);
Tab L = Tab("p", "xxx","x");
wind->changeActiveTabTo(1);
wind->print();
Tab R = Tab("vvv", "rr","ee");
wind->newTab(F);
wind->newTab(H);
Tab V = Tab("sss", "kkk","dd");
wind->closeTab();
Tab E = Tab("t", "uu","j");
wind->newTab(E);
Tab P = Tab("q", "tt","l");
wind->newTab(J);
wind->isEmpty();
wind->getActiveTab();
Tab X = Tab("mm", "dd","zz");
wind->moveActiveTabTo(5);
Tab Z = Tab("x", "d","ccc");
wind->getActiveTab();
wind->getActiveTab();
Tab Q = Tab("lll", "fff","ff");
wind->moveActiveTabTo(1);
Tab O = Tab("t", "l","n");
wind->changeActiveTabTo(3);
wind->print();
Tab K = Tab("iii", "z","o");
wind->print();
wind->print();
wind->changeActiveTabTo(5);
wind->isEmpty();
wind->moveActiveTabTo(4);
wind->getActiveTab();
wind->getActiveTab();
wind->getActiveTab();
wind->getActiveTab();
wind->isEmpty();
wind->moveActiveTabTo(0);
wind->newTab(K);
wind->newTab(L);
wind->getActiveTab();
wind->changeActiveTabTo(0);
wind->newTab(M);
Tab A = Tab("hhh", "e","p");
wind->getActiveTab();
wind->changeActiveTabTo(3);
wind->print();
wind->changeActiveTabTo(0);
wind->changeActiveTabTo(0);
wind->isEmpty();
wind->getActiveTab();
wind->closeTab();
wind->newTab(A);
wind->getActiveTab();
wind->changeActiveTabTo(3);
wind->print();
wind->changeActiveTabTo(2);
wind->getActiveTab();
wind->changeActiveTabTo(0);
wind->getActiveTab();
wind->print();
wind->print();
Tab C = Tab("www", "j","oo");
wind->closeTab();
Tab B = Tab("uuu", "a","ggg");
wind->newTab(B);
wind->moveActiveTabTo(5);
Tab G = Tab("q", "uu","bbb");
wind->isEmpty();
wind->getActiveTab();
wind->print();
wind->moveActiveTabTo(1);
wind->print();
wind->print();
wind->getActiveTab();
wind->newTab(C);
wind->print();
wind->print();
wind->print();
wind->newTab(G);
wind->moveActiveTabTo(2);
wind->getActiveTab();
wind->closeTab();
wind->newTab(N);
wind->isEmpty();
wind->closeTab();
wind->newTab(O);
wind->newTab(P);
wind->moveActiveTabTo(3);
wind->newTab(Q);
wind->closeTab();
wind->print();
wind->print();
wind->isEmpty();
wind->getActiveTab();
wind->closeTab();
wind->newTab(R);
wind->moveActiveTabTo(5);
wind->print();
wind->newTab(S);
delete wind;
}
	return 0;
}