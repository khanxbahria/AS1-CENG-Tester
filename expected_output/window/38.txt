The active tab is tab 0




The active tab is tab 2




ttt
cc
p

ff
aaa
hhh

The active tab is tab 2




ttt
cc
p

ff
aaa
hhh

The active tab is tab 0




The active tab is tab 3




z
oo
n

s
z
bbb

uu
t
j

The active tab is tab 3




z
oo
n

s
z
bbb

uu
t
j

The active tab is tab 3




z
oo
n

s
z
bbb

uu
t
j

The active tab is tab 3
uu
t
j

www
j
qq

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 3
www
j
qq

e
hhh
p

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 0
www
j
qq

e
hhh
p

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 0
www
j
qq

e
hhh
p

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 5
e
hhh
p

z
iii
o

xxx
p
x





z
oo
n

a
uuu
ggg

s
z
bbb

The active tab is tab 1
e
hhh
p

a
uuu
ggg

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 1
e
hhh
p

a
uuu
ggg

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 2
e
hhh
p

a
uuu
ggg

j
www
oo

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 2
e
hhh
p

a
uuu
ggg

j
www
oo

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 2
e
hhh
p

a
uuu
ggg

j
www
oo

z
iii
o

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 4
e
hhh
p

a
uuu
ggg

j
www
oo

tt
q
l

z
iii
o

l
t
n

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 4
e
hhh
p

a
uuu
ggg

j
www
oo

tt
q
l

z
iii
o

l
t
n

xxx
p
x





z
oo
n

s
z
bbb

The active tab is tab 5
e
hhh
p

a
uuu
ggg

j
www
oo

tt
q
l

l
t
n

rr
vvv
ee

xxx
p
x





z
oo
n

s
z
bbb

